webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -35,29 +49,97 @@ function GithubItems(props) {
+
+        if (item.reviews) {
+            const reviewerUsers = [];
+            const filteredReviews = item.reviews.filter((v) => {
+                if (!reviewerUsers.includes(v.user.login)) {
+                    reviewerUsers.push(v.user.login);
+                    return true;
+                }
+                return false;
+            });
+
+            const approved = filteredReviews.reduce((accum, cur) => {
+                if (cur.state === 'APPROVED') {
+                    return accum + 1;
+                }
+                return accum;
+            }, 0);
+            const changesRequested = filteredReviews.reduce((accum, cur) => {
+                if (cur.state === 'CHANGES_REQUESTED') {
+                    return accum + 1;
+                }
+                return accum;
+            }, 0);
+
+            const totalReviewers = item.reviewers + item.reviews.length;
+            if (totalReviewers > 0) {
+                let reviewName;
+                if (totalReviewers === 1) {
+                    reviewName = 'review';
+                } else {
+                    reviewName = 'reviews';
+                }
+                reviews = (<span>{approved} out of {totalReviewers} {reviewName} complete.</span>);
+            }
+
+            if (changesRequested > 0) {
+                changes = (<div style={{...iconStyle, fill: '#c11b28'}}><DotIcon/></div>);
+            }
+        }
+
+        let status = '';
+
+        // Status images pasted directly from GitHub. Change to our own version when styles are decided.
+        if (item.status) {
+            switch (item.status) {
+            case 'success':
+                status = (<div style={{...iconStyle, fill: '#2b9643'}}><TickIcon/></div>);
+                break;
+            case 'pending':
+                status = (<div style={{...iconStyle, fill: '#c59e17'}}><DotIcon/></div>);
+                break;
+            default:
+                status = (<div style={{...iconStyle, fill: '#c11b28'}}><CrossIcon/></div>);
+            }
+        }
+
         return (
             <div
                 key={item.id}
                 style={style.container}
             >
                 <div>
                     <strong>
-                        {title}
+                        {title}{status}
                     </strong>
                 </div>
+                <div>
+                    <strong><i className='fa fa-code-fork'/> #{item.number}</strong> <span className='light'>{repoName}</span>
+                </div>
                 <GithubLabels labels={item.labels}/>
                 <div
                     className='light'
                     style={style.subtitle}
                 >
-                    {userName ? 'Created by ' + userName + ' ' : ''}
-                    {'at ' + repoName + '.'}
+                    {'Opened ' + formatTimeSince(item.created_at) + ' ago'}
+                    {userName ? ' by ' + userName : '.'}
```

Comment:
```suggestion
                    {userName && ' by ' + userName}{'.'}
```

I think the punctuation should be consistent here whether there is a username or not.